.env.example

```diff
@@ -1,3 +1,3 @@
-frontify_domain=demo.frontify.com # change to own domain
-frontify_access_token=vi7.......zrsg # obtain on {domain}/api/developer/token
+frontify_domain=demo.frontify.com
+frontify_access_token=vi7.......zrsg
```

Comment:
For me this actually did cause an issue, and I've no idea why. I first thought it was dotenv caching something so set override=True when loading the dotenv, to ensure that it wasn't using a system variable, but the only way I was able to fix it was removing it from the dotenv file.

I removed it from the example since I figured if someone is using this collection of scripts, they may not be as experienced technically and so would copy the example including comments which may cause them issues that they don't know how to debug

Reply:
ok, fair enough. Then let's keep this as is - you've given instructions on how to get the token in other readme files.